scripts/uploader.rb

```diff
@@ -19,13 +19,12 @@ def self.upload(directory, type)
     end
 
     def initialize(directory, type)
-      @container = "release"
       @directory = directory
       @type      = type
     end
 
     def run
-      login
+      rackspace_client.login
```

Comment:
I love the idea of having pluggable uploaders.  This way if we have to change again in the future we just write up another plugin.  So that being said, can remove any notion of there being client specifics?

Let's say we have a clients array as an accessor, then if we put a dummy login method into the DO plugin, then this line can be `clients.each(&:login)` and it removes client specifics.

Reply:
Done